examples/rllib.py

```diff
@@ -19,27 +27,52 @@
 
 # Add custom metrics to your tensorboard using these callbacks
 # see: https://ray.readthedocs.io/en/latest/rllib-training.html#callbacks-and-custom-metrics
-def on_episode_start(info):
-    episode = info["episode"]
-    episode.user_data["ego_speed"] = []
+class Callbacks(DefaultCallbacks):
+    def on_episode_start(
+        self,
+        *,
+        worker: RolloutWorker,
+        base_env: BaseEnv,
+        policies: Dict[PolicyID, Policy],
+        episode: MultiAgentEpisode,
+        env_index: int,
+        **kwargs,
+    ):
 
+        episode.user_data["ego_speed"] = []
 
-def on_episode_step(info):
-    episode = info["episode"]
-    single_agent_id = list(episode._agent_to_last_obs)[0]
-    obs = episode.last_raw_obs_for(single_agent_id)
-    episode.user_data["ego_speed"].append(obs["speed"])
+    def on_episode_step(
+        self,
```

Comment:
`self` and `*` seem not to be used?

Reply:
Sure, yes, the * is not needed here it was auto-generated from the base method.